packages/admin-app/src/components/AppBar/AppBar.tsx

```diff
@@ -57,7 +58,9 @@ export const AppBar: React.FunctionComponent<AppBarProps> = ({ title, loggedIn }
         <MaterialAppBar position="static" title={title}>
             <Toolbar className={classes.toolbar}>
                 <Box className={classes.logoContainer}>
-                    <ElectionGuardLogo className={classes.logo} />
+                    <a href={routeIds.home}>
+                        <ElectionGuardLogo className={classes.logo} />
+                    </a>
```

Comment:
Nitpick: This could be in isolated smaller PR. Won't affect approval. Just for future Pr's.